generators/app/templates/hooks/pre-commit

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
We probably could, but I don't think it's worth it. 1) bash is just gross. 2) all code is gross when you don't write it every day 3) I've used this code on Optimas and TPB, but I've never had to actually look at it.